data-reconciliation-app/src/models/summary-record.ts

```diff
@@ -0,0 +1,145 @@
+import { MINIMUM_VOTES_THRESHOLD } from "../utils/constants";
+import { ServiceResult } from "../utils/service-result";
+import { DataAttributeType } from "./data-record";
+import { ReconciledRecord } from "./reconciled-record";
+
+export enum SummaryGroupStatus {
+  NotEnoughVotes = "NOT_ENOUGH_VOTES",
+  LackOfConsensus = "LACK_OF_CONSENSUS",
+  InConsensus = "IN_CONSENSUS",
+}
+
+export enum SummaryStatus {
+  Majority = "Majority",
+  Minority = "Minority",
+}
+
+export interface SummaryRecordProps {
+  key: string;
+  value: DataAttributeType;
+  type: string;
+  status: SummaryStatus;
+  groupStatus: SummaryGroupStatus;
+  membersInAgreementCount: number;
+  membersInDisagreementCount: number;
+  votesCount: number;
```

Comment:
is this the same thing as total_votes_count that would be added later as governance?